src/MediatR.Extensions.FluentValidation.AspNetCore/MediatR.Extensions.FluentValidation.AspNetCore.csproj

```diff
@@ -21,10 +21,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FluentValidation" Version="8.2.2" />
-    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.2.2" />
-    <PackageReference Include="MediatR" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.0" />
+    <PackageReference Include="FluentValidation" Version="8.6.2" />
+    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.6.2" />
+    <PackageReference Include="MediatR" Version="8.0.1" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.5" />
```

Comment:
Is it needed to change versions? It limit availability for others.

Reply:
Reverted